### PR TITLE
Update README for Ez-Schedule

### PR DESCRIPTION
**Pull request by BigheartSmallbrain**
Add README.md which contains an overview and draft ui mockup

### CODE REVIEW

**Review by anthonynmh — COMMENTED** [review]
UI mockup may need an ending time. Other than that, the rest looks good

**Comment by lhy-hoyin** [comment]
I see that you are committing your PPP here and also in #32. Perhaps you should only commit via one of the PR only.

File: docs/team/bigheartsmallbrain.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+Zhang Yuquan's Project Portfolio Page
+---
+
+### <span style="color:orange"> Project: EzSchedule </span>
```
Comment (lhy-hoyin) [inline docs/team/bigheartsmallbrain.md:5]: Should it be "Ez-Schedule" (have the dash) instead?

**Review by lhy-hoyin — APPROVED** [review]
LGTM

**Comment by lhy-hoyin** [comment]
Upon merging, go to #5 and tick the README checkbox

**Pull request merged**